WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -219,15 +219,21 @@ public class BlockEditorScreen: ScreenObject {
     }
 
     private func chooseFromDevice() throws -> MediaPickerAlbumScreen {
-        let imageDeviceButton = app.sheets.buttons["Choose from device"] // Uses a localized string
+        let imageDeviceButton = app.buttons["Choose from device"] // Uses a localized string
 
         imageDeviceButton.tap()
 
         // Allow access to device media
         app.tap() // trigger the media permissions alert handler
 
-        return try MediaPickerAlbumListScreen()
-                    .selectAlbum(atIndex: 0)
+        if #available(iOS 16.0, *), XCUIDevice.isPhone {
+            // After the UI interuption monitor handler is called, the "Recents" album is automatically tapped.
+            // But this only happens on iPhone, not iPad...
```

Comment:
It turns out, this "automatic album selection" is device specific. For example, it occurs on iPhone 13 mini, but not iPhone SE, not sure about other simulators. This now looks very much like a bug in Xcode beta, I'm thinking removing this code, and let affected tests fail (for now), and re-evaluate in next beta.